perma_web/perma/templates/search.html

```diff
@@ -1,5 +1,6 @@
 {% extends "base-responsive.html" %}
-{% load file_exists pipeline %}
+{% load file_exists %}
```

Comment:
Observation: my pending pull request also alters this {% load %} line, and others like it. So whichever one we merge second will need to be rebased and tweaked.